restylers.yaml

```diff
@@ -185,6 +185,20 @@
   documentation:
     - https://github.com/ndmitchell/hlint#readme
     - https://github.com/restyled-io/restyled.io/wiki/Common-Errors:-HLint
+- enabled: false
+  name: isort
+  image: restyled/restyler-isort:4.3.21
+  command:
+    - isort
+  arguments: []
+  include:
+    - "**/*.py"
+  interpreters:
+    - python
+  supports_arg_sep: true
+  supports_multiple_paths: true
+  documentation:
+    - https://github.com/timothycrosley/isort
```

Comment:
Just FYI (in case of future contributions), you don't have to update this file, it's automatically handled during release. For example, the `image` here is out of date with the `v`-version change. I'll take care of it after merge though!

Reply:
Thanks for your explanation @pbrisbin, I'll take it into account in future contributions. 😉